README.md

```diff
@@ -229,6 +229,7 @@ The devices below are fully feature supported by OpenRazer, which means all avai
 | Razer Goliathus                               |  1532:0C01  |
 | Razer Goliathus Extended                      |  1532:0C02  |
 | Razer Firefly v2                              |  1532:0C04  |
+| Razer Firefly v2 Pro                          |  1532:0C08  |
```

Comment:
Please keep this list sorted, so move it one below

Reply:
i did it!